azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -482,13 +482,56 @@
     }
 
     /**
+     * The virtual machine definition stage with availability set.
+     *
+     * @param <T> The virtual machine definition in creatable stage.
+     */
+    interface DefinitionWithAvailabilitySet<T extends DefinitionCreatable> {
+        /**
+         * Specifies the name of the availability set to create, the virtual machine will be part of
+         * this availability set.
```

Comment:
i'd shorten: "Specifies the name of a new availability set to put the virtual machine in."